### ISSUE TITLE
items as promise

### ISSUE THREAD

**Issue by fdu-axiometrics** [valor-software/ng2-select#270]
Our data for selection is too big to put in an array on client side. 
We have an api to search for data, like typeahead with promise.

Is it possible to have this?

Thanks!

**Comment by linusbrolin** [comment]
Isn't it better to use another solution if the data array is so big?
A select box for huge arrays is not exactly user friendly.
A suggestion would be to use a clickable list or table and a paginator instead, if the array is that huge.

**Comment by Gillardo** [comment]
Doesnt @fdu-axiometrics mean he wants to be able to load data on demand like you could with ui-select??  I would like this feature to, any code samples would be appreciated

**Comment by fdu-axiometrics** [comment]
yeah. that's what I mean. Load data dynamically from server, not from hard coded arrays..

**Comment by 0cv** [comment]
interested too ...

**Comment by denbezrukov** [comment]
any solutions?
